src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
can you also do macosx + voiceover? if that works, can add a section about aria-live to the accessibility wiki in adapt? then we can all know how to use it and that it's fine to use.